sandbox-open-imis/charts/db/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: open-imis-secret
```

Comment:
Why not use variables from values file here as well like from deployment?

Reply:
https://github.com/GovStackWorkingGroup/sandbox-open-imis/pull/1/commits/59675dc95eb546ced4c6089484c6a1bebd8f7fb1